src/ansys/aedt/core/workflows/project/version_manager.py

```diff
@@ -0,0 +1,527 @@
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+VERSION = "0.3.0"
+
+import os
+from pathlib import Path
+import platform
+import shutil
+import subprocess
+import sys
+import tkinter as tk
+from tkinter import filedialog
+from tkinter import messagebox
+from tkinter import ttk
+import webbrowser
+import xml.etree.ElementTree as ET
+import zipfile
+
+import ansys.aedt.core as pyaedt
+from ansys.aedt.core.workflows.customize_automation_tab import add_script_to_menu
+from ansys.aedt.core.workflows.customize_automation_tab import available_toolkits
+from ansys.aedt.core.workflows.misc import get_aedt_version
+from ansys.aedt.core.workflows.misc import get_port
+from ansys.aedt.core.workflows.misc import get_process_id
+import pyedb
+import requests
+
+DISCLAIMER = (
+    "This script will download and install certain third-party software and/or "
+    "open-source software (collectively, 'Third-Party Software'). Such Third-Party "
+    "Software is subject to separate terms and conditions and not the terms of your "
+    "Ansys software license agreement. Ansys does not warrant or support such "
+    "Third-Party Software.\n"
+    "Do you want to proceed ?"
+)
+
+
+def is_git_installed():
+    try:
+        # Run the command `git --version`
+        result = subprocess.run(["git", "--version"], stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
+
+        # Check if the command ran successfully
+        if result.returncode == 0:
+            print(f"Git is installed: {result.stdout.strip()}")
+            return True
+        else:
+            print("Git is not installed.")
+            return False
+    except FileNotFoundError:
+        print("Git is not installed.")
+        return False
+
+
+def get_latest_version(package_name):
+    response = requests.get(f"https://pypi.org/pypi/{package_name}/json")
+    if response.status_code == 200:
+        data = response.json()
+        return data["info"]["version"]
+    else:
+        return "Unknown"
+
+
+class VersionManager:
+    TITLE = "Version Manager {}".format(VERSION)
+    USER_GUIDE = "https://github.com/ansys-internal/pyaedt_version_manager"
```

Comment:
This is an internal website. I think you could point to the pyaedt documentation (once it is built)

Reply:
Yes, please remove references to this org.